@ecomplus/storefront-template/package-lock.json

```diff
@@ -0,0 +1,1541 @@
+{
```

Comment:
Teve algo errado no seu teste local @matheusgnreis , esse arquivo não deveria existir aqui

Reply:
Meu rodou tranquilo

Reply:
Rodar vai kkk

Reply:
Mas vai cagar as dependências depois e dar conflito na PR, como tá dando aí

Reply:
`npm i` no root do repositório apenas, foi só isso que você fez?

Reply:
Inclusive eu não tô conseguindo dar merge por causa disso kkkk

Reply:
Foi

Reply:
Dei npm i, tentei npm run serve ai deu ruim, vi que tinha que instalar yarn, ai fiz isso global pra poder fazer isso com qualquer repo que eu usar e ai consegui rodar. Qualquer coisa abro nova PR, dou npm i no root

Reply:
Não, tinha que instalar yarn nenhum não kkk

Reply:
É só `npm i` no root do repositório e mais nada na real, se isso não funcionou talvez tenha algo de errado no seu ambiente, se for Windows é provável que tenha mesmo

Reply:
Depois do `npm i` no root do repo o `npm run serve` também no root do repo vai funcionar